src/routes/posts.js

```diff
@@ -78,9 +78,34 @@ const deletePost = async (req, res, next) => {
     }
 }
 
+// UPDATE AN ARTICLE
+
+const updatePost = async (req, res, next) => {
+    const { id } = req.params // WHERE
+    const { title, content, image } = req.body // SET
```

Comment:
i think someone can also decide an existing article so might have to be added here. I saw you have it in your manual test as part of your sample data though

Reply:
oh okay, i will add that too